skimage/_shared/gpu/convolve.py

```diff
@@ -0,0 +1,32 @@
+from scipy import ndimage as ndi
+import numpy as np
+import skimage._shared.gpu
+#setup the imports, attempt gputools, if not, fallback instantly
+#
+
+'''
+  mode_str = {"constant":"CLK_ADDRESS_CLAMP",
+                "wrap":"CLK_ADDRESS_REPEAT"}
+                "wrap":"CLK_ADDRESS_REPEAT",
+                "nearest":"CLK_ADDRESS_CLAMP_TO_EDGE",
+                "reflect":"CLK_ADDRESS_MIRRORED_REPEAT"}
+'''
```

Comment:
@Dalordish this string is totally useless, as far as I can tell. =)

Reply:
Yup, was an artifact from some mucking around that is irrelevant, fixed!